metadata-ingestion/src/datahub/ingestion/source/ldap.py

```diff
@@ -86,14 +98,27 @@ def set_cookie(
     return bool(cookie)
 
 
-class LDAPSourceConfig(ConfigModel):
+class LDAPSourceStatefulIngestionConfig(StatefulStaleMetadataRemovalConfig):
```

Comment:
this entire class is no longer necessary!

Reply:
I think I got it, I can remove  LDAPSourceStatefulIngestionConfig now and use StatefulStaleMetadataRemovalConfig instead.